docs/ClusterSetup.md

```diff
@@ -183,6 +183,9 @@ ledgerDirectories=data/bookkeeper/ledgers
 
 # Point to local ZK quorum
 zkServers=zk1.example.com:2181,zk2.example.com:2181,zk3.example.com:2181
+
+# Change the ledger manager type
+ledgerManagerType=hierarchical
```

Comment:
Main issue is that the minimum viable config might not be the best config for production use. In the broker config we can fix that, and already the conf file should have the same values that are taken as default. In BookKeeper config, however, we want to have different values from the BK defaults.